hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/scm/TestStorageContainerManager.java

```diff
@@ -311,7 +324,7 @@ public void testBlockDeletionTransactions() throws Exception {
         OzoneTestUtils.closeContainers(keyInfo.getKeyLocationVersions(),
```

Comment:
I see that the other tests have similar logic (such as: `testBlockDeletingThrottling`, it close `Container` too). Is it possible to have the same bugs.

Reply:
Since this test also passes in repeated runs, I would not want to change anything in this test as of now. Kindly re-review.